bot.py

```diff
@@ -396,6 +427,7 @@ async def pause(self, message, _command_content):
 
         self.voice_client.pause()
         logging.info("Paused media for user %s", message.author)
+        await self.attempt_disconnect()
         await message.add_reaction(MusicBot.REACTION_EMOJI)
```

Comment:
Swap these two lines, so the reaction comes immediately, not 5 minutes later.
```suggestion
        await message.add_reaction(MusicBot.REACTION_EMOJI)
        await self.attempt_disconnect()
```

Reply:
fixed